Web/web-example-metamask.html

```diff
@@ -1,6 +1,7 @@
 <html>
+
 <head>
-    <script src="../../dist/streamr-client.web.js"></script>
+    <script src="https://unpkg.com/streamr-client@5.4.4/streamr-client.web.js"></script>
```

Comment:
Replaced with cdn link

Reply:
Does unpkg support @ latest? I suspect this repo will be easier to maintain if it is always pulling the latest streamr clients

Reply:
@mxinburritos